kernel: timer: enable tickless kernel test for XTENSA

spoorthik opened the pull request:
Added CONFIG_XTENSA to testcase.yaml file to enable
    tickless kernel test case for XTENSA.

Signed-off-by: Spoorthi K <spoorthi.k@intel.com>

spoorthik committed:
kernel: timer: enable tickless kernel test for XTENSA

    Added CONFIG_XTENSA to testcase.yaml file to enable
    tickless kernel test case for XTENSA.

Signed-off-by: Spoorthi <spoorthi.k@intel.com>

SavinayDharmappa commented:
shippable is failing below error:

 UC2 Signed-off-by: must have a full name
use  following command to set your full name  git config --global user.name "<full name>"

lpereira commented on tests/kernel/timer/timer_api/testcase.yaml:
```diff
@@ -4,5 +4,5 @@ tests:
       build_only: true
       extra_args: CONF_FILE="prj_tickless.conf"
-      filter: CONFIG_BOARD_QEMU_X86
+      filter: (CONFIG_BOARD_QEMU_X86 or CONFIG_XTENSA)
```
Shouldn't this be CONFIG_BOARD_QEMU_XTENSA instead?

spoorthik replied:
To run on ESP32 "CONFIG_XTENSA" is required. Or should we make the application to run only on QEMU, to make it generic?

spoorthik committed:
Merge branch 'master' of https://github.com/zephyrproject-rtos/zephyr

spoorthik committed:
kernel: timer: enable tickless kernel test for XTENSA

    Added CONFIG_XTENSA to testcase.yaml file to enable
    tickless kernel test case for XTENSA.

Signed-off-by: Spoorthi K <spoorthi.k@intel.com>

spoorthik committed:
Merge branch 'master' of https://github.com/spoorthik/zephyr
